lib/omniauth/bigcommerce/version.rb

```diff
@@ -15,6 +15,6 @@
 #
 module OmniAuth
   module BigCommerce
-    VERSION = '0.3.3.pre'.freeze
+    VERSION = '0.3.4.pre'.freeze
```

Comment:
I don't think you need to bump this - https://github.com/bigcommerce/omniauth-bigcommerce/releases/tag/v0.3.2 is the latest released version, and there's no reason to skip 0.3.3.

Reply:
👍